Jenkinsfile

```diff
@@ -28,6 +28,7 @@ pipeline {
         stage('checkout') {
             steps {
                 checkout scm
+                sh 'echo $(git log --format=format:%H -n1) > actual-commit.file'
```

Comment:
Actually, jenkins expose a GIT_COMMIT variable which you can use instead perhaps?